### PR TITLE
fix: remove various offix hacks

### PR DESCRIPTION
**Pull request by wtrocki**
Contains @darahayes changes

### CODE REVIEW

File: client/src/config/clientConfig.ts [hunk 1]
```diff
@@ -82,10 +82,17 @@ const cache =  new InMemoryCache({
...
 });
 
-export const clientConfig = {
+export const clientConfig: ApolloOfflineClientOptions = {
   link: authLink.concat(splitLink),
   cache: cache,
   conflictListener: new ConflictLogger(),
   mutationCacheUpdates: globalCacheUpdates,
-  networkStatus: new WebNetworkStatus()
+  inputMapper: {
```
Comment (wtrocki) [inline client/src/config/clientConfig.ts]: Any reason why we do not use this clever implementation of traverse library?

Comment (wtrocki) [reply client/src/config/clientConfig.ts]: Actually that might be better. We can evern have it in offix as CRUDInputMapper

**Comment by kingsleyzissou** [comment]
@wtrocki fixed the build issue

**Comment by wtrocki** [comment]
Amazing. Let's merge it so we can do tests on master

**Pull request merged**